truss/templates/server/common/truss_server.py

```diff
@@ -166,6 +166,21 @@ async def predict(
                 headers=response_headers,
             )
 
+    async def schema(self, model_name: str) -> Dict:
+        model: ModelWrapper = self._safe_lookup_model(model_name)
+
+        if model.truss_schema is None:
+            # If there is not a TrussSchema, we return a 404.
+            raise HTTPException(status_code=404, detail="No schema found")
```

Comment:
This seems retry-able (like we set truss_schema to Null at init, but set it at load). i wonder if there is a more clear error code here so that users can retry later.

Reply:
I still don't know if a 404 is correct here.

Reply:
@bolasim is what you have in mind here only returning 404 after `load` completes? And maybe before load, it can return 503, which would indicate there should be a retry.

Reply:
that would be consistent with what we do today for the readiness probe, which feels right now to me.

Reply:
cool, done 👍